lms/config/resources.py

```diff
@@ -1,5 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not directly related to this PR, but in Python 3, files without a coding declaration [default to UTF-8](https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations). Is there still any reason to include this?

Reply:
:+1: There's lots of these in the LMS app, I guess we should remove them all